src/api.ts

```diff
@@ -334,3 +345,26 @@ export class MirrordAPI {
         });
     }
 }
+
+
+/** 
+* Updates the global feedback counter. When it hits `FEDBACK_COUNTER_REVIEW_AFTER` mirrord runs, 
+* displays a message asking the user to like mirrord.
+*/ 
+function tickFeedbackCounter() {
+    const counter = parseInt(globalContext.globalState.get(FEEDBACK_COUNTER) ?? '0') || 0;
```

Comment:
Why is the `|| 0` necessary?

Reply:
Same answer as the `update` stringify question.

Reply:
Not needed as well.